packages/api/src/middleware/hardcoded-nodes.ts

```diff
@@ -22,17 +39,17 @@ export default function hardcodedNodes({
   }
   return (req, res, next) => {
     if (!req.getBroadcasters) {
-      req.getBroadcasters = async () => broadcasters;
+      req.getBroadcasters = async () => broadcasters as NodeAddress[];
     }
     if (!req.getOrchestrators) {
-      req.getOrchestrators = async () => orchestrators;
+      req.getOrchestrators = async () => orchestrators as NodeAddress[];
     }
     if (!req.getIngest) {
-      req.getIngest = async () => ingest;
+      req.getIngest = async () => ingest as Ingest[];
     }
-    if (!req.prices) {
-      req.getPrices = async () => prices;
+    if (!req.getPrices) {
```

Comment:
Hm, this check must have never worked with the wrong name there. I also don't really know what it's for. @yondonfu @kyriediculous?

Reply:
I think it really has never worked indeed, always overwriting what was in the request. I think it wasn't really an issue though because no other middleware overwrites that property. References in repo: https://github.com/livepeer/livepeer-com/search?q=getPrices

The `kubernetes` one actually [only sets `getBroadcasters` and `getOrchestrators`](https://github.com/livepeer/livepeer-com/blob/bd7528e04a91310fe2baa833d4504df0f96dde70/packages/api/src/middleware/kubernetes.ts#L107) so I'm guessing we only use the "hardcoded" ones configured on `LP_PRICES` env [in livepeer-infra](https://github.com/livepeer/livepeer-infra/blob/f78b771069043f14bc6fa79145cfdb392aee46fd/values/prod/livepeer.values.yaml#L5-L10)